Cargo.toml

```diff
@@ -26,10 +26,11 @@ doc = ["file-lock", "unprivileged", "tokio-runtime"]
 futures-util = { version = "0.3", features = ["sink"] }
 futures-channel = { version = "0.3", features = ["sink"] }
 async-trait = "0.1"
+cstr = "0.2"
 libc = "0.2"
 bincode = "1.3"
 serde = { version = "1.0", features = ["derive"] }
-nix = "0.20"
+nix = { git = "https://github.com/nix-rust/nix.git", rev = "7317e0a3871c58ce3e29de8ee26ead10bdcb979e" }
```

Comment:
I remember crate.io doesn't allow crate's dependency use git version, all dependencies need specify a version

Reply:
That's right; it won't allow you to publish a crate when Cargo.toml contains a git revision.  So you can merge the PR, but you won't be able to publish until after Nix does.